gaj23/face-it
# Gaj/fix/capitalization

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 

This branch contains two fixes
- an addition to anchor tags to protect our app when a user navigates away from our page
- the capitalization of `Components` inside of github/git


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.

We were trying to deploy our app when we came across this problem. Despite the changes being reflected inside our local devices, git didn't recognize our changes. We had to go through the CLI to make this case-sensitive change occur in git. 

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

See our `Category` file to view the `rel` attribute addition. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

The console should no longer issue warnings about missing `rel` tags. 

Additionally, there should be no `build error` through the Vercel bot. 

### Steps To Take
> Any questions and possible steps moving forward.

Check Vercel to confirm deployment/build is possible.


#### Checklist:
